pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -418,7 +420,7 @@ func configTimerTask(getconfigCtx *getconfigContext, handleChannel chan interfac
 				warningTime, errorTime)
 
 		case <-stillRunning.C:
-			if getconfigCtx.configProcessingSkipFlag {
+			if getconfigCtx.configProcessingSkipFlagReboot {
 				log.Noticef("config processing skip flag set")
```

Comment:
Here is makes sense to log if it is anything but ConfigOk. 
Note that you can define a String member function for configProcessingRetval so can log the values using %s.

Reply:
Added in a dedicated commit.